lib/dialyzer/src/typer.erl

```diff
@@ -31,566 +34,41 @@
 -define(SHOW, show).
 -define(SHOW_EXPORTED, show_exported).
 -define(ANNOTATE, annotate).
+-define(ANNOTATE_IN_PLACE, annotate_in_place).
```

Comment:
If it's not too much trouble, can you split this addition into its own commit?

Reply:
Hey John! I went ahead and did pretty much the opposite. We decided to remove for now the `annotate_in_place` mode to make this PR easier to follow and more concise and so that we can try it more thoroughly with our rebar3 plugin.

We can add it back if you want to, but otherwise expect another PR to add it after this one has been done with :)